src/Umbraco.Web.UI.Client/src/views/users/views/users/users.controller.js

```diff
@@ -583,7 +583,10 @@
                         dateVal = moment(user.lastLoginDate, "YYYY-MM-DD HH:mm:ss");
                     }
 
-                    user.formattedLastLogin = dateVal.format("MMMM Do YYYY, HH:mm");
+                    // get current backoffice user and format date
+                    userService.getCurrentUser().then(function (currentUser) {
+                        user.formattedLastLogin = dateVal.locale(currentUser.locale).format("LLL");
+                    });
```

Comment:
I could also get current user by using the following instead of userService, but I don't think that would be safe, e.g. when using paging:
```
var currentUser = users.filter(function (u) { return u.isCurrentUser === true; })[0];
user.formattedLastLogin = dateVal.locale(currentUser.culture).format("LLL");
```